doc/roadmap.rst

```diff
@@ -144,19 +143,6 @@ bottom.
      :issue:`6929`
    * Callbacks or a similar system would facilitate logging and early stopping
 
-#. Use scipy BLAS Cython bindings
-
-   * This will make it possible to get rid of our partial copy of suboptimal
-     Atlas C-routines. :issue:`11638`
-   * This should speed up the Windows and Linux wheels
-
-#. Allow fine-grained parallelism in cython
-
-   * Now that we do not use fork-based multiprocessing in joblib anymore it's
-     possible to use the prange / openmp thread management which makes it
-     possible to have very efficient thread-based parallelism at the Cython
-     level. Example with K-Means: :issue:`11950`
```

Comment:
#11950 is not merged?

Reply:
The entry isn't specifically about #11950, it's about using openmp which we do now with #12807

Reply:
Fair enough, thanks. I agree that we can have this PR in 0.21.